src/main/java/com/ua/accommodation/dto/user/UserResponseDto.java

```diff
@@ -15,5 +15,5 @@ public class UserResponseDto {
     private String lastName;
     @ToString.Exclude
     @EqualsAndHashCode.Exclude
-    private Set<Role> roles = new HashSet<>();
+    private Set<Role.RoleName> roles = new HashSet<>();
```

Comment:
you can import - "import static com.ua.accommodation.model.Role.RoleName;" and use just RoleName